inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/source/ServerMessageHandler.java

```diff
@@ -525,7 +529,8 @@ private void responsePackage(ChannelHandlerContext ctx, Map<String, String> comm
             Channel remoteChannel,
             SocketAddress remoteSocketAddress,
             MsgType msgType) throws Exception {
-        if (!commonAttrMap.containsKey("isAck") || "true".equals(commonAttrMap.get("isAck"))) {
+        if (!commonAttrMap.containsKey(AttributeConstants.MESSAGE_IS_ACK)
+                || "true".equals(commonAttrMap.get(AttributeConstants.MESSAGE_IS_ACK))) {
```

Comment:
It is better. And 
String isAck = commonAttrMap.getOrDefault(AttributeConstants.MESSAGE_IS_ACK, Boolean.TRUE.toString());
if (Boolean.TRUE.toString().equals(isAck)) {

Reply:
done